.travis.yml

```diff
@@ -20,20 +23,23 @@ jobs:
     - stage: Tests
       script: make test-ci
 
-#    - stage: E2E Tests
-#      script: make test-e2e
-#      before_script:
-#        # Download kubectl, which is a requirement for using minikube and running e2e tests.
-#        - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
-#        # Download minikube.
-#        - curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
-#        # Start minikube.
-#        - sudo minikube start --vm-driver=none --bootstrapper=localkube --kubernetes-version=${KUBERNETES_VERSION}
-#        # Update the kubeconfig to use the minikube cluster.
-#        - minikube update-context
-#        # Wait for Kubernetes to be up and ready.
-#        # TODO: Check for /healthz
```

Comment:
The `/healthz` endpoint is [disabled by default for Minikube](https://github.com/kubernetes/minikube/blob/dcb5c2cc5077d9963ce96833490cce73bd225feb/pkg/localkube/proxy.go#L67), so we can't use to check clusters health. It could possibly be enabled, but I believe checking is node ready is enough.

I also added check waiting for kube-dns to become ready, so we're sure we have DNS working.